### DISCUSSION TITLE
Removing My Account

### DISCUSSION THREAD

**Discussion by ghost** [jvangeld/ME-GIS#30] [Announcements]
Due to GitHub's new requirement of 2 Factor Autorization, I will be deleting my account sometime in the next month. ME-GIS is hosted on my account. So if anyone would like to take over the project I can transfer it to you.

Once upon a time, you needed a phone number to access the internet because you had to dial in with a modem. We thought we left those bad old days in the past when we moved to broadband. But the elites who run this country are scared that they might not know your every move. Or that their bots might get confused between an identity that is not verifiably the same as another identity.

What if those elites learned to live with ignorance and risk, like the rest of us? What if they decided to rule their people with justice, so that they don't have to fear their own nation? The wicked flee when no one pursues, but the righteous are as bold as a lion.

**Comment by andrewheiss** [comment]
I'd be happy to take ownership!

**Reply by ghost** [reply]
Perfect, let's make that happen.

**Comment by ghost** [comment]
Thanks for taking this over, @andrewheiss. I hope it keeps getting better.
